Software-Engineering/L5-Principal-Software-Engineer.md

```diff
@@ -1,97 +1,124 @@
+- I am an expert at managing scope, finding ways to help my teams consistently deliver the high value to customers at speed.
+- I hold myself and my teams to a high standard of quality, with a specific focus on reliability, usability, and functionality.
 - I am responsive to changes in product priorities, keeping multiple teams focused on important, high-value efforts.
 - I inform product strategy by helping senior leadership understand the organisation's engineering capabilities and technical strategy.
+- I help people from other departments, primarily support, customer success, revenue, and marketing, when my expertise will deliver better outcomes for our customers.
 
 <details>
 <summary>Examples</summary>
 
 - I helped a team pivot effectively in response to a direction change.
-- I helped a team find a faster path to market.
+- I helped a team find a faster path to market/customer value/adoption, helping them make acceptable tradeoffs in scope, time, and quality.
 - I identified a risk where multiple teams would make life harder for each other, got them aligned, and accelerated their deliveries.
-- I contributed to the Octopus blog, explaining a lesson we learned as a way to reinforce the learning and to attract customers and potential future employees.
-- I identified a risk to our product strategy and influenced a change to mitigate that risk.
+- I contributed to the Octopus blog, a webinar, or a conference, explaining a lesson we learned as a way to reinforce the learning and to attract customers and potential future employees.
+- I participated actively and willingly in discovery activities, like customer research calls, and generated insights that improved overall customer outcomes.
+- I noticed one of my teams getting bogged down in delivery. I helped them tease apart their scope and sequence their work to get themselves unblocked and get back to delivering value more quickly.
+- I regularly tried the software delivered by my teams (I "tasted the food") and provided feedback where the reliability, usability, or functionality didn't meet our standard.
+- I identified a risk in our product strategy or delivery that required a difficult tradeoff and influenced a change to mitigate that risk.
 - I built an influential case to change direction/priority with a focus on promoting customer success.
+- I helped transform a customer at risk of churn into a case study. I did this by joining several calls with the customer, understanding the big picture, diagnosing the critical problems, making the customer feel heard, and ultimately delivering on our commitments.
+- I helped close-win a complex deal where my contributions provided the buyer with confidence in the functionality and quality of the services we offer today, and/or confidence in the product direction aligned to my group/teams. I also identified some areas we can improve in our current service offerings that will simplify similar deals in the future.
```

Comment:
```suggestion
- I helped transform a customer at risk of churn into a case study. I did this by joining several calls with the customer, understanding the big picture, diagnosing the critical problems, making the customer feel heard, and ultimately delivering on our commitments.
- I helped close-win a complex deal where my contributions provided the buyer with confidence in the functionality and quality of the services we offer today, and/or confidence in the product direction aligned to my group/teams. I also identified some areas we can improve in our current service offerings that will simplify similar deals in the future.
```

💭 Clarifying: I agree these are great outcomes... but they feel more like outcomes of a team (or a leadership team) working together. Is there a way we can be more concrete and specific about the PE contribution here (in contrast to other roles)? Examples:

- I joined several calls with a customer who was at risk of churn. I used my listening skills and my insight into the technical capabilities of our product to add clarity to the discussion. This enabled us to transform this customer into a case study.

- I was pulled into a complex deal negotiation. I used my insight into the functionality and quality of the services we offer today, and my group's product direction, to provide the buyer with confidence. This enabled the deal to get to Closed-Won.